axelrod/plot.py

```diff
@@ -1,4 +1,6 @@
-import numpy
+
+import numpy # for numpy.linalg.linalg.LinAlgError
```

Comment:
Why not be explicit and directly import LinAlgError
```
from numpy.linalg import LinAlgError
```
Then you can drop the comment and don't need to import all of numpy for a single reference.